GET-10

MattFontaine opened the pull request:
Photo Gallery

michaelconnor00 committed:
Add .gitignore file with idea included.

michaelconnor00 committed:
Add node_modules to gitignore.

michaelconnor00 committed:
Add the explore page to the app.

michaelconnor00 committed:
Add controller for the explore page.

michaelconnor00 committed:
Add stock html page for explore.

michaelconnor00 committed:
Add explore to side bar.

michaelconnor00 committed:
Remove node_modules.

michaelconnor00 committed:
Merge commit '6312a8cff7bb161ca93a86aeb2ae3519d91fce61' into GET-7

# Conflicts:
#	.gitignore
#	node_modules/ansi-regex/package.json
#	node_modules/ansi-styles/package.json
#	node_modules/archy/package.json
#	node_modules/array-find-index/package.json
#	node_modules/balanced-match/package.json
#	node_modules/bower/package.json
#	node_modules/brace-expansion/package.json
#	node_modules/builtin-modules/package.json
#	node_modules/camelcase-keys/package.json
#	node_modules/camelcase/package.json
#	node_modules/chalk/package.json
#	node_modules/clone-stats/package.json
#	node_modules/clone/package.json
#	node_modules/concat-map/package.json
#	node_modules/core-util-is/package.json
#	node_modules/currently-unhandled/package.json
#	node_modules/dateformat/package.json
#	node_modules/decamelize/package.json
#	node_modules/defaults/package.json
#	node_modules/deprecated/package.json
#	node_modules/duplexer2/package.json
#	node_modules/end-of-stream/package.json
#	node_modules/error-ex/package.json
#	node_modules/escape-string-regexp/package.json
#	node_modules/extend/package.json
#	node_modules/find-index/package.json
#	node_modules/find-up/package.json
#	node_modules/findup-sync/package.json
#	node_modules/first-chunk-stream/package.json
#	node_modules/flagged-respawn/package.json
#	node_modules/gaze/package.json
#	node_modules/get-stdin/package.json
#	node_modules/glob-stream/node_modules/glob/package.json
#	node_modules/glob-stream/node_modules/minimatch/package.json
#	node_modules/glob-stream/node_modules/readable-stream/package.json
#	node_modules/glob-stream/node_modules/through2/package.json
#	node_modules/glob-stream/package.json
#	node_modules/glob-watcher/package.json
#	node_modules/glob/package.json
#	node_modules/glob2base/package.json
#	node_modules/globule/node_modules/glob/package.json
#	node_modules/globule/node_modules/graceful-fs/package.json
#	node_modules/globule/node_modules/inherits/package.json
#	node_modules/globule/node_modules/minimatch/package.json
#	node_modules/globule/package.json
#	node_modules/graceful-fs/package.json
#	node_modules/gulp-util/README.md
#	node_modules/gulp-util/index.js
#	node_modules/gulp-util/lib/PluginError.js
#	node_modules/gulp-util/lib/buffer.js
#	node_modules/gulp-util/lib/env.js
#	node_modules/gulp-util/lib/isBuffer.js
#	node_modules/gulp-util/lib/isNull.js
#	node_modules/gulp-util/lib/isStream.js
#	node_modules/gulp-util/lib/log.js
#	node_modules/gulp-util/lib/template.js
#	node_modules/gulp-util/package.json
#	node_modules/gulp/package.json
#	node_modules/has-ansi/package.json
#	node_modules/hosted-git-info/package.json
#	node_modules/indent-string/package.json
#	node_modules/inflight/package.json
#	node_modules/inherits/package.json
#	node_modules/interpret/package.json
#	node_modules/is-arrayish/package.json
#	node_modules/is-builtin-module/package.json
#	node_modules/is-finite/package.json
#	node_modules/is-utf8/package.json
#	node_modules/isarray/package.json
#	node_modules/liftoff/package.json
#	node_modules/load-json-file/package.json
#	node_modules/lodash._reinterpolate/LICENSE.txt
#	node_modules/lodash._reinterpolate/README.md
#	node_modules/lodash._reinterpolate/index.js
#	node_modules/lodash._reinterpolate/package.json
#	node_modules/lodash.escape/README.md
#	node_modules/lodash.escape/index.js
#	node_modules/lodash.escape/package.json
#	node_modules/lodash.keys/README.md
#	node_modules/lodash.keys/index.js
#	node_modules/lodash.keys/package.json
#	node_modules/lodash.template/README.md
#	node_modules/lodash.template/index.js
#	node_modules/lodash.template/package.json
#	node_modules/lodash.templatesettings/README.md
#	node_modules/lodash.templatesettings/index.js
#	node_modules/lodash.templatesettings/package.json
#	node_modules/lodash/package.json
#	node_modules/loud-rejection/package.json
#	node_modules/lru-cache/package.json
#	node_modules/map-obj/package.json
#	node_modules/meow/node_modules/object-assign/package.json
#	node_modules/meow/package.json
#	node_modules/minimatch/package.json
#	node_modules/minimist/package.json
#	node_modules/mkdirp/node_modules/minimist/package.json
#	node_modules/mkdirp/package.json
#	node_modules/multipipe/package.json
#	node_modules/normalize-package-data/package.json
#	node_modules/number-is-nan/package.json
#	node_modules/once/package.json
#	node_modules/orchestrator/package.json
#	node_modules/ordered-read-streams/package.json
#	node_modules/os-homedir/package.json
#	node_modules/parse-json/package.json
#	node_modules/path-exists/package.json
#	node_modules/path-is-absolute/package.json
#	node_modules/path-type/package.json
#	node_modules/pify/package.json
#	node_modules/pinkie-promise/package.json
#	node_modules/pinkie/package.json
#	node_modules/pretty-hrtime/package.json
#	node_modules/process-nextick-args/package.json
#	node_modules/read-pkg-up/package.json
#	node_modules/read-pkg/package.json
#	node_modules/readable-stream/package.json
#	node_modules/rechoir/package.json
#	node_modules/redent/package.json
#	node_modules/repeating/package.json
#	node_modules/resolve/package.json
#	node_modules/semver/package.json
#	node_modules/sequencify/package.json
#	node_modules/sigmund/package.json
#	node_modules/signal-exit/package.json
#	node_modules/spdx-correct/package.json
#	node_modules/spdx-exceptions/package.json
#	node_modules/spdx-expression-parse/package.json
#	node_modules/spdx-license-ids/package.json
#	node_modules/stream-consume/package.json
#	node_modules/string_decoder/package.json
#	node_modules/strip-ansi/package.json
#	node_modules/strip-bom/package.json
#	node_modules/strip-indent/package.json
#	node_modules/supports-color/package.json
#	node_modules/through2/node_modules/isarray/package.json
#	node_modules/through2/node_modules/readable-stream/package.json
#	node_modules/through2/package.json
#	node_modules/tildify/package.json
#	node_modules/trim-newlines/package.json
#	node_modules/unique-stream/package.json
#	node_modules/user-home/package.json
#	node_modules/util-deprecate/package.json
#	node_modules/v8flags/package.json
#	node_modules/validate-npm-package-license/package.json
#	node_modules/vinyl-fs/node_modules/clone/package.json
#	node_modules/vinyl-fs/node_modules/graceful-fs/package.json
#	node_modules/vinyl-fs/node_modules/readable-stream/package.json
#	node_modules/vinyl-fs/node_modules/strip-bom/package.json
#	node_modules/vinyl-fs/node_modules/through2/package.json
#	node_modules/vinyl-fs/node_modules/vinyl/package.json
#	node_modules/vinyl-fs/package.json
#	node_modules/vinyl/LICENSE
#	node_modules/vinyl/README.md
#	node_modules/vinyl/index.js
#	node_modules/vinyl/lib/cloneBuffer.js
#	node_modules/vinyl/lib/inspectStream.js
#	node_modules/vinyl/lib/isBuffer.js
#	node_modules/vinyl/lib/isNull.js
#	node_modules/vinyl/package.json
#	node_modules/wrappy/package.json
#	node_modules/xtend/package.json

michaelconnor00 committed:
Add app.explore and app.map to routing.

michaelconnor00 committed:
Reformat to 2 space indent. Remove ExploreCtrl to separate controller.

michaelconnor00 committed:
Add exploreController.

michaelconnor00 committed:
Update explore.html with list of maps.

michaelconnor00 committed:
Create map.html for individual maps.

Commit:
Added a photo-slider that acts as the homepage before the app (changed app flow)

qkmaxware committed:
gulp task for injection of css into the index. Requires gulp-inject to be installed.

Commit:
Added favourites page

qkmaxware committed:
Settings page template.

michaelconnor00 committed:
Add controller for maps.

michaelconnor00 committed:
Add CartoDB scripts.

michaelconnor00 committed:
Temporary comment out map controller.

michaelconnor00 committed:
Comment out map html.

michaelconnor00 committed:
Add mapController to index.html.

michaelconnor00 committed:
Remove comments from controller.

michaelconnor00 committed:
Remove map creation from controller.

qkmaxware committed:
Settings page template.

michaelconnor00 committed:
Delete node modules.

michaelconnor00 committed:
Merge pull request #5 from artbabu/GET-12

GET-12 Added a photo-slider that acts as the homepage before the app (change…

qkmaxware committed:
resolved conflicts for merge of GET-11

qkmaxware committed:
Update controllers.js

qkmaxware committed:
Update app.js

qkmaxware committed:
Merge pull request #8 from artbabu/GET-11

Get 11

Commit:
Merge pull request #4 from artbabu/GET-15

GET - 15

michaelconnor00 committed:
Added trail data in geojson format.

artbabu committed:
Code Cleanup

qkmaxware committed:
Contact us page

qkmaxware committed:
Added general links

qkmaxware committed:
Added general links

Commit:
Changed 4 space indents to 2 space indents

Commit:
Merge remote-tracking branch 'origin/development' into GET-9

# Conflicts:
#	www/js/app.js
#	www/js/controllers.js
#	www/templates/menu.html

Commit:
Fixed semicolon-related error

Commit:
changed from 4 space indents to 2 spaced indents

Commit:
changed from 4 space indents to 2 spaced indents

michaelconnor00 committed:
Merge pull request #9 from artbabu/GET-9

GET-9 Added favourites page

michaelconnor00 committed:
Create mapService for fetching trail data.

michaelconnor00 committed:
Merge branch 'development' into GET-31

michaelconnor00 committed:
Add starter.factories to app.js.

michaelconnor00 committed:
Add cartodb service to index.

michaelconnor00 committed:
Add dependency list to call.

michaelconnor00 committed:
Add console logs for testing.

michaelconnor00 committed:
Refactored to return promise.

michaelconnor00 committed:
Add cartoDB.js to index.

michaelconnor00 committed:
Refactor controller to use service.

qkmaxware committed:
made all remote images local

Commit:
Added initial explorer page

Commit:
Added initial explorer page

artbabu committed:
Removed node_modules from repository

artbabu committed:
Merge branch 'GET-7' of github.com:artbabu/GetLost into GET-7

artbabu committed:
Merge branch 'GET-7' into development

# Conflicts:
#	www/js/app.js
#	www/js/controllers.js

artbabu committed:
merged commit from GET-7

artbabu committed:
Merge branch 'development' of github.com:artbabu/GetLost into development

Commit:
Merge remote-tracking branch 'origin/GET-data' into GET-18

qkmaxware committed:
Fixing contact us page.

qkmaxware committed:
Merge pull request #11 from artbabu/GET-20

Get 20 merged to development.

qkmaxware commented on www/js/app.js:
```diff
@@ -25,9 +25,14 @@ angular.module('starter', ['ionic', 'starter.controllers'])
 .config(function($stateProvider, $urlRouterProvider) {
   $stateProvider
 
```
I cannot find your controller for the gallery page.

Commit:
Added search bar and segmented control plugins

Commit:
Included searchbar and segmented control links & scripts

Commit:
Added Explore state and controller

Commit:
Added searchbar and segmented control to Explore page

Commit:
Reformatted current resources

Commit:
Fixed typo in geojson place name

Commit:
Added script tags for services.js and for all geojson data.

Commit:
Added service file and a service to get data from the geojson resource files

Note: resource files are hardcoded as I couldn't find a way to deal with directives

Commit:
Updated ExploreCtrl to deal with geojsonService

Commit:
Added starter.services module to requirements

Commit:
Refactored name of ItemCtrl.

Commit:
Added integration with ExploreCtrl to display geojson data.

Commit:
Rename Explore.html to explore.html

Commit:
Commented out all but one resource file

Commit:
Added search bar and filter button control

Commit:
Added a ngMaterial requires to the module

Commit:
Explore page initial card-based look, with functioning searchbar and segmented control

Commit:
Added new libraries for searchbar, segmented control, and tile-grids

Commit:
Readded mainslide styling

Commit:
Updated Library script locations to point to locations on the web like Github

Commit:
Deleted actual library code as it is accessable on the web

Commit:
Fixed Segment buttons not displaying words

michaelconnor00 committed:
Change service to factory.

michaelconnor00 committed:
Merge branch 'development' into GET-31

# Conflicts:
#	www/index.html

michaelconnor00 committed:
Add layer name as a constant.

michaelconnor00 committed:
Remove MapService from controllers.

michaelconnor00 committed:
Add MapService to mapController.

michaelconnor00 committed:
Update explore template for new data.

Commit:
Changed Card aesthetics

Commit:
Changed online resource of segment control to a local resource lib folder

Commit:
Added refresher function to ExploreCtrl, changed startCtrl to go to app.explore off the bat

Commit:
Updated resource references

Commit:
Added proper refresher for dynamic data

Commit:
Updated online references

Commit:
Added android platform

Commit:
Added android platform and gave access to all online domains so as to use online libraries

Commit:
Gave access to only online libraries used as opposed to the whole web

Commit:
Merge branch 'development' into GET-18

# Conflicts:
#	www/index.html
#	www/js/app.js
#	www/templates/explore.html

Commit:
Removed duplicate ExploreCtrl

michaelconnor00 committed:
Add notes about adding future functions.

Commit:
Made segmentedSelectedIndex variable only available in controller scope

Commit:
Merge pull request #16 from artbabu/GET-31

Get 31 - Add CartoDB service for accessing data

michaelconnor00 committed:
Merge branch 'development' into GET-18

# Conflicts:
#	www/index.html
#	www/js/app.js
#	www/js/mapController.js
#	www/templates/explore.html

michaelconnor00 committed:
Merge pull request #17 from artbabu/GET-18

Get-18: Build Explore Page
